react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const pokeRender = pokedex.map(pokemon =>
+    <li key={pokemon.number.toString()}>
```

Comment:
Since the `number` property is already a string in this case, we don't need to use `toString` here